pirania-app/files/usr/libexec/rpcd/pirania-app

```diff
@@ -27,6 +27,12 @@ local function shell(command)
     return result
 end
 
+local function show_governance(msg)
+    local result = {}
+    result = json.decode(shell('cat /etc/pirania/governance.json'))
```

Comment:
to read a json you don't need to do a shell call to do a cat, you can open the file for read:
```
    local file = io.open( path, "r" )
    local contents = file:read( "*a" )
    local result = Json.decode(contents);
```